core/docs/Changelog.md

```diff
@@ -2,4 +2,31 @@
 
 ## 0.1.0 (March 2023)
 
-* Initial release
+Also see [streamly-core-0.1.0 API Changelog](/core/docs/ApiChangelogs/0.1.0.txt) or
+https://hackage.haskell.org/package/streamly-core-0.1.0/docs/docs/ApiChangelogs/0.1.0.txt
+
+`streamly` package is split into two packages, (1) `streamly-core` that
+has only GHC boot library depdendecies, and (2) `streamly` that contains
+higher level operations (including concurrent ones) with additional
+dependencies.
+
+* Moved the following modules from `streamly` package to the
```

Comment:
I remember you telling me to remove this? You readded this?

Reply:
Did I? Any move should be mentioned in both origin and destination.